style.css

```diff
@@ -344,39 +344,39 @@ body.lock {
     position: relative;
     height: 330px;
 }
-.img1 {
+.about-part__img-container_img1 {
     position: absolute;
     left: 0;
     top: 140px;
     z-index: 10;
     height: 150px;
 }
-.img2 {
+.about-part__img-container_img2 {
     position: absolute;
     right: 0; 
     top: 0; 
     height: 150px;
 }
 .title {
-    font-size: 35px;
+    font-size: 11vw;
     letter-spacing: 8px;
 }
 .subtitle {
     color: var(--additional-color);
     font-family: var(--main-text-font);
-    font-size: 12px;
+    font-size: 3.5vw;
```

Comment:
This is funny now 12px is 3.5vw  :D 
So far we got 
12px === 3.7vw
12px === 4vw
12px === 3.5vw

Reply:
This is a subjective assessment, after revaluation

Reply:
Changed all sizes to 3.5 vw to be consistent with each other

Reply:
Went back to px